app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -651,7 +651,11 @@ public void find_new_raw_curve() {
             BgReading latest_entry = BgReading.lastNoSenssor();
             ra = 0;
             rb = 0;
-            rc = latest_entry.age_adjusted_raw_value;
+            if (latest_entry != null) {
```

Comment:
In any event, Im all about checking to make sure things are not null, better safe than sorry!